velox/expression/SimpleFunctionAdapter.h

```diff
@@ -223,6 +224,10 @@ class SimpleFunctionAdapter : public VectorFunction {
           "Variadic args can only be used as the last argument to a function.");
       auto oneReader = VectorReader<arg_at<POSITION>>(packed, POSITION);
 
+      if constexpr (FUNC::udf_has_callNullFree) {
```

Comment:
do we not need  oneReader.setChildrenMayHaveNulls(); here?

Reply:
Originally I thought not because this is only for Variadic types, but I should be calling it on the underlying readers, so yes.